### PR TITLE
Add sentry integration

### PR DESCRIPTION
**Pull request by jsugarman**
Add sentry integration

### CODE REVIEW

File: fee_calculator/settings/docker.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from .base import *  # noqa
 
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY')
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
```
Comment (jsugarman) [inline fee_calculator/settings/docker.py]: @ibrechin is this needed?

File: fee_calculator/apps/api/filters.py [hunk 2]
```diff
@@ -57,8 +57,8 @@ def filter(self, qs, value):
         if self.distinct:
             qs = qs.distinct()
         qs = self.get_method(qs)(
-            Q(**{'%s__isnull' % self.field_name: True}) |
-            Q(**{'%s__%s' % (self.field_name, lookup): value})
+            Q(**{'%s__isnull' % self.name: True}) |
```
Comment (jsugarman) [inline fee_calculator/apps/api/filters.py]: @ibrechin could adding this error back in be causing a complete internal server error on staging https://laa-fee-calculator-staging.apps.cloud-platform-live-0.k8s.integration.dsd.io/

File: .gitattribute [hunk 3]
```diff
@@ -1 +0,0 @@
-django-secret.yaml filter=git-crypt diff=git-crypt
```
Comment (jsugarman) [inline .gitattribute]: @ibrechin we removed this file but could it be needed?

File: fee_calculator/settings/base.py [hunk 4]
```diff
@@ -28,6 +28,7 @@ def location(folder):
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', 'MY-SECRET')
+SENTRY_DSN = os.environ.get('SENTRY_DSN', 'MY-SENTRY-DSN')
```
Comment (ibrechin) [inline fee_calculator/settings/base.py]: This line is not necessary.

File: fee_calculator/settings/docker.py (hunk 1 above)
Comment (ibrechin) [inline fee_calculator/settings/docker.py]: This line is not necessary.

**Pull request merged**